cdap-ui/app/cdap/components/FieldLevelLineage/OperationsModal/index.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import {Actions} from 'components/FieldLevelLineage/store/Store';
+import { Modal, ModalHeader, ModalBody } from 'reactstrap';
+import LoadingSVG from 'components/LoadingSVG';
+import ModalContent from 'components/FieldLevelLineage/OperationsModal/ModalContent';
+
+require('./OperationsModal.scss');
+
+function OperationsModalView({showOperations, loading, closeModal, fieldName}) {
+  if (!showOperations) { return null; }
+
+  const loadingIndicator = (
+    <div className="loading-container text-xs-center">
+      <LoadingSVG />
+    </div>
+  );
+
+  return (
+    <Modal
+      isOpen={true}
+      toggle={closeModal}
+      size="lg"
+      backdrop="static"
+      zIndex="1061"
+      className="field-level-lineage-modal"
+    >
+      <ModalHeader>
+        <span>
+          Operations for field {`'${fieldName}'`}
+        </span>
+
+        <div
+          className="close-section float-xs-right"
+          onClick={closeModal}
+        >
+          <span className="fa fa-times" />
```

Comment:
nit: use `IconSVG` and `icon-close`

Reply:
ahh, i was trying to use `icon-times` and couldn't find it.. lol